CHANGES.txt

```diff
@@ -4,11 +4,8 @@ CHANGES
 0.15.0 (2017-xx-xx)
 -------------------
 
--
+- Enable async support feature available in Jinja 2.9 #152
```

Comment:
do we update this file manually referencing issue number (e.g. 152 here)?

Reply:
yes manually, please share if you know better way.